modules/ROOT/pages/transaction-management.adoc

```diff
@@ -146,7 +145,122 @@ When running in a transactional scope, the entire process must run in the same t
 
 Any other router or scope (for example, Foreach) continues with the transaction and its behavior is not modified.
 
+== Error Handling
+
+When an error occurs during a transaction, your application must be able to handle the error and continue with the transaction or do a rollback.
+
+There are two types of Error Handlers, which behave differently when an error is raised during a transaction:
+
+* On Error Propagate
+** If the `on-error-propagate` is inside the Error Handler corresponding to the component that began the transaction:
++
+The transaction is rolled back without executing the inner processors of the On Error Propagate Scope. This means that the inner processors do not run within the transaction.
+
+** If the `on-error-propagate` is inside an inner element that did not start the transaction:
++
+The transaction is not rolled back and the inner processors of the `on-error-propagate` run within the transaction. Remember that some scopes and routers behave differently when executing within a transaction.
+
+* On Error Continue
++
+The error is considered managed, the transaction remains active and is able to commit. The inner processors of the On Error Continue run within the transaction.
+
+=== Example: Error Raised in Try Scope
+
+In this first example, the message source (`jms:listener`) starts a transaction, which is continued by the `try` Scope (`transactionalAction` is set to default, `INDIFFERENT`).
+
+When the error is raised, it is handled by the `error-handler` of the `try` Scope first. The error is handled by an `on-error-propagate`, but the transaction is not rolled back because it was not the `try` Scope the one that started the transaction. Only the error handler of the flow can rollback the transaction because it was initiated at flow level (from its message source).
+
+The `logger` inside the `on-error-propagate` runs within the transaction. The error is then propagated to the flow's `error-handler`, which handles it with an `on-error-continue`. The `logger` inside the `on-error-continue` runs within the transaction. Finally, the transaction is committed.
+
+[source,xml,linenums]
+----
+<flow name="someFlowWithTx">
+	<jms:listener config-ref="JMS_Config" destination="test.in" transactionalAction="ALWAYS_BEGIN"/>
+	<!-- Processors -->
+	<try>
+		<raise-error type="APP:SOME"/>
+		<error-handler>
+			<on-error-propagate>
+				<logger message="Fails"/>
+			</on-error-propagate>
+		</error-handler>
+	</try>
+	<error-handler>
+		<on-error-continue>
+			<logger message="continue with tx"/>
+		</on-error-continue>
+	</error-handler>
+</flow>
+----
+
+If we replace the `on-error-continue` with an `on-error-propagate` in the example above, then the transaction is rolled back, and the logger inside this `on-error-propagate` does not run within the transaction.
```

Comment:
Please check that from the review form @elrodro83 I've modified this examples to use processors that currently manages transactions, to make this clearer